commcare_export/__init__.py

```diff
@@ -21,7 +21,7 @@ def logger_name_from_filepath(filepath):
     return (
         relative_path.
         replace('/', '.').
-        replace('.py', '')
+        strip('.py')
```

Comment:
The `strip()` method strips any of the given characters from either side of a string.
```
>>> foo = 'pyppy.excel_query.py'
>>> foo.strip('.py')
'excel_quer'
>>> foo.rstrip('.py')
'pyppy.excel_quer'
```

So `rstrip()` is closer to what you want, but will still be too aggressive with `excel_query.py` because "excel_query" ends in a "y".

What you're really looking for is
```
>>> import re
>>> re.sub(r'\.py$', '', foo)
'pyppy.excel_query'
```

Reply:
@kaapstorm 
Woah, thanks for illuminating this. I was definitely not aware of the full blast radius that `strip()` had. I always thought that it should match the string _exactly_ in order for the function to strip it...good reminder to double check those functions you thought you knew...

Reply:
I added a new test with the example you gave...just to be sure...